codalab/worker/main.py

```diff
@@ -205,7 +210,7 @@ def main():
     worker.start()
 
 
-def parse_cpuset_args(arg):
+def cpuset_args(arg):
```

Comment:
Can we keep the `parse_cpuset_args` name to be consistent with the `parse_size` functions?